tests/features/ibm-bamoe/ibm-bamoe-kieserver.feature

```diff
@@ -76,15 +86,15 @@ Feature: RHPAM KIE Server configuration tests
     Then container log should contain -Dorg.kie.executor.retry.count=40
      And container log should contain - Retries per Request: 40
 
-  Scenario: KIECLOUD-122 - Enable JMS for RHDM and RHPAM, verify if the JMS is the default executor and jms transacted is false
+  Scenario: KIECLOUD-122 - Enable JMS for IBM BAMOE, verify if the JMS is the default executor and jms transacted is false
```

Comment:
not sure if this is referencing JIRA KIECLOUD-122 summary text or not? in case of JIRA summary we should not change/replace it IMHO